config/300-repositories.yaml

```diff
@@ -170,7 +170,11 @@ spec:
                         name:
                           type: string
                           description: "The secret name"
-
+                repo_list_to_scope_token:
```

Comment:
maybe github_app_token_scope_reos it's a bit long but at least people understand it straight away what it does

Reply:
updated to `github_app_token_scope_repos`